pootle/settings/50-project.conf

```diff
@@ -121,3 +121,5 @@ AUTH_USER_MODEL = 'accounts.User'
 COMMENTS_APP = "pootle_comment"
 
 POOTLE_EXPORT_VIEW_LIMIT = 10000
+
+POOTLE_STATS_REFRESH_ATTEMPTS_COUNT = 2
```

Comment:
This is missing an explanatory comment here as @dwaynebailey suggested. Please also pay attention to explain properly the `= 0` scenario.